CanvasCore/CanvasCore/Helm/Extensions/HelmSplitViewControllerExtensions.swift

```diff
@@ -16,7 +16,7 @@
 
 import UIKit
 
-public extension HelmSplitViewController {
+extension HelmSplitViewController {
```

Comment:
This `public` causes the `description`'s `open` to be a warning, but changing that to `public` is an error.

Reply:
Nice. Yeah `public extension` isn't a thing.